ui/src/components/ApprovalsTracker/index.tsx

```diff
@@ -146,6 +149,19 @@ const ApprovalsTracker: React.FC<Props> = (props): React.ReactElement => {
         [props.publication.createdBy, user?.id]
     );
 
+    const displayArrayOfAffiliations = (affiliations: Interfaces.MappedOrcidAffiliation[]) => {
```

Comment:
whenever displaying these affiliations, they need to be sorted. I have created a helper function for this in /lib/helpers.tsx.
<img width="906" alt="image" src="https://github.com/JiscSD/octopus/assets/48569671/abc1e61f-db68-46f0-8538-c85b6a2e5052">